README.md

```diff
@@ -153,6 +153,7 @@ More information on contributing and the general code of conduct for discussion
 | Word to PDF                              | [Word to PDF](https://github.com/DhanushNehru/Python-Scripts/tree/master/Word%20to%20PDF%20converter)                                                    | A Python script to convert an MS Word file to a PDF file.                                                                                                         |
 | Youtube Downloader                       | [Youtube Downloader](https://github.com/DhanushNehru/Python-Scripts/tree/master/Youtube%20Downloader)                                                    | Downloads any video from [YouTube](https://youtube.com) in video or audio format!                                                                                 |
 | Youtube Playlist Info Scraper            | [Youtube Playlist Info Scraper](https://github.com/DhanushNehru/Python-Scripts/tree/master/Youtube%20Playlist%20Info%20Scraper)                          | This python module retrieve information about a YouTube playlist in json format using playlist link.                                                              |
+| Mood based youtube song generator        | [Mood based youtube song generator](https://github.com/DhanushNehru/Python-Scripts/tree/master/Mood%20based%20youtube%20song%20generator)                | This Python script fetches a random song from YouTube based on your mood input and opens it in your default web browser.                                                              |
```

Comment:
This is changes should be here. Not the rest. Please remove others and put only this change. Thanks

Reply:
fixed, thank you for the review